Updated readme & design

sampoearas opened the pull request:
- Updated readme layout
- Updated Install Steps with IOM install
- Updated design of webpage (colors, font, text placeholders)
- Updated screenshot with new design
- Tested project on Aras v11.12-15, v12.0

sampoearas committed:
Updated readme & design

- Updated readme layout
- Updated Install Steps with IOM install
- Updated design of webpage
- Updated screenshot with new design
- Tested project on Aras v12.0

EliJDonahue commented on README.md:
```diff
@@ -5,33 +5,53 @@ This is a very simple Visual Studio Web Application that includes several text b
 
-![Customized Dashboard](./Screenshots/PR%20Web%20Page%20Filled%20In.PNG)
-*The PR Web Page with log in credentials and PR data filled out*
+![Customized Dashboard](./Screenshots/PR-WebPage2.PNG)
```
This url doesn't work. GitHub is case-sensitive, and the actual file name has a lower case file extension.

sampoearas replied:
Fixed extension

EliJDonahue reviewed (changes requested):
There are a lot of new files added that aren't needed and shouldn't be included in the repository, like the sqlite3 folder, .vs folder, all of the dlls in the bin folder, and everything in the obj folder. Look up a .gitignore template for this type of Visual Studio project, add it to your local repo, and ignore/untrack these files.

EliJDonahue commented on README.md:
```diff
@@ -5,33 +5,53 @@ This is a very simple Visual Studio Web Application that includes several text b
 2. Start the External_Input_Example project.
-3. Wait for the the web page to load
+   1. Select the browser you want to run it in.
+3. Wait for the the web page to load.
```
Repeated "the". Please fix.

sampoearas replied:
Fixed grammar.

sampoearas committed:
Add gitignore file

sampoearas committed:
Create .gitignore

sampoearas committed:
Delete VisualStudio.gitignore

sampoearas committed:
Apply .gitignore prefs

Revert ".gitignore is now working"

This reverts commit f0fc00081cabff191de9ab573fa15a1bd851f52d.

Update .gitignore file

- bypass Aras DLLs
Apply .gitignore prefs

Revert "Apply .gitignore prefs"

This reverts commit e8621e60fb0b13e8087ee3225ed3d89b844d5b5b.

Update gitignore


Apply .gitignore prefs

Revert "Apply .gitignore prefs"

This reverts commit 457f5341e94b2082dc3ddce304ac5bb20e647f15.

update gitignore


Apply .gitignore prefs

update gitignore


Apply .gitignore prefs

update gitignore


Apply .gitignore prefs

sampoearas committed:
Fixed Screenshot ref

sampoearas committed:
Update README.md

removed double "the" in usage

sampoearas commented:
Changes made. gitignore added.
